test/v2/core/minter.spec.js

```diff
@@ -0,0 +1,211 @@
+const {ethers} = require("hardhat");
+const {expect} = require("chai");
+const {parseEther} = require("ethers/lib/utils");
+
+describe("SharedDepositMinterV2", () => {
+  let sgEth, paymentSplitter, minter, withdrawals, wsgEth, deployer, alice, multiSig;
+
+  beforeEach(async () => {
+    const [owner, addr1, addr2] = await ethers.getSigners();
+
+    const SgETH = await ethers.getContractFactory("SgETH");
+    sgEth = await SgETH.deploy([]);
+    await sgEth.deployed();
+
+    deployer = owner;
+    alice = addr1;
+    multiSig = addr2;
+
+    MINTER_ROLE = await sgEth.MINTER();
+
+    // deploy sgeth
+    const WSGETH = await ethers.getContractFactory("WSGETH");
+    wsgEth = await WSGETH.deploy(sgEth.address, 24 * 60 * 60);
+    await wsgEth.deployed();
+
+    const splitterAddresses = [deployer.address, multiSig.address, wsgEth.address];
+    const splitterValues = [6, 3, 31];
+
+    const PaymentSplitter = await ethers.getContractFactory("PaymentSplitter");
+    paymentSplitter = await PaymentSplitter.deploy(splitterAddresses, splitterValues);
+    await paymentSplitter.deployed();
+
+    const rolloverVirtual = "1080000000000000000";
+    const vETH2Addr = "0x898bad2774eb97cf6b94605677f43b41871410b1";
+
+    const Withdrawals = await ethers.getContractFactory("Withdrawals");
+    withdrawals = await Withdrawals.deploy(vETH2Addr, rolloverVirtual);
+    await withdrawals.deployed();
+
+    const numValidators = 1000;
+    const adminFee = 0;
+
+    const FeeCalc = await ethers.getContractFactory("FeeCalc");
```

Comment:
try to use the zero address for this like the deploy instead. 
the idea was to flesh out and launch a fee calc contract for configuring fees later. but it should default to running internal logic if the passed in address is address(0)